src/js/components/InfiniteScroll/InfiniteScroll.js

```diff
@@ -114,8 +114,15 @@ class InfiniteScroll extends PureComponent {
       const endRect = findDOMNode(
         this.lastPageItemRef.current,
       ).getBoundingClientRect();
-      /* eslint-enable react/no-find-dom-node */
-      const nextPageHeight = endRect.y + endRect.height - beginRect.y;
+      let nextPageHeight;
+      // IE & Edge
+      if (document.documentMode || /Edge/.test(window.navigator.userAgent)) {
```

Comment:
Instead of adding the browser check, can we just change to use `.top` instead of `.y` always?

Reply:
Makes sense. Thanks!!